### DISCUSSION TITLE
A - B repetition

### DISCUSSION THREAD

**Discussion by nikorion** [deckerst/aves#962] [Ideas]
Hello,
I really like this app and would like to get rid of MX player (and VLC), but the A-B repeat feature is very useful for me. 
If it were possible to add this to Aves, the app would be just perfect in my eyes :p

**Comment by deckerst** [comment]
Pardon my ignorance, but what is the A-B repeat feature?

**Reply by nikorion** [reply]
The feature consists of selecting a piece of a video during playback and then looping it.
In VLC on Windows, for example :
![2024-03-30 02_29_11-Paramètres](https://github.com/deckerst/aves/assets/164088448/1b5aed13-6e66-4aa5-8c8e-f7531cea4e1d)
![2024-03-30 02_29_46-Toolbars Editor](https://github.com/deckerst/aves/assets/164088448/a1f57f28-8270-484f-b691-c72d7a413076)

**Comment by deckerst** [comment]
I see, thanks. Out of curiosity, how is that feature useful to you?

**Reply by nikorion** [reply]
I use it to improve my English pronunciation (I'm a French speaker) by looping certain phrases and then repeating them.

But it can also be used to analyze an artistic or sporting performance, to train by observing a movement, or to easily adjust the moment when subtitles should appear. Maybe it can also be used for presentations or demonstrations.

Ps : If you find the motivation to do this and it doesn't take up too much of your time, that's cool. Otherwise, it's no problem, your app is still great ;-)

**Reply by deckerst** [reply]
<img width="200" src="https://github.com/deckerst/aves/assets/13898333/572c5a3c-8349-4be7-b2fe-9b152c6e085a"/>

J'allais tout de même pas manquer l'opportunité d'aider un francophone voulant améliorer son anglais ! ^^

**Reply by nikorion** [reply]
Wonderful, thanks a lot ! :D
